GovUk.Frontend.AspNetCore.Extensions/wwwroot/govuk/govuk-validation.js

```diff
@@ -30,10 +30,17 @@ function createGovUkValidator() {
     },
 
     /**
-     * Ensures the page title is prefixed with 'Error: ' when there is at least one .govuk-error-message displayed
+     * Ensures the page title is prefixed with the data-govuk-error-prefix attribute, or 'Error: ' when there is at least one .govuk-error-message displayed
      */
-    updateTitle: function () {
-      const prefix = "Error: ";
+      updateTitle: function () {
+        const getPrefix = document.querySelector("[data-govuk-error-prefix]")
+        let prefix = ""
```

Comment:
There's inconsistent spacing here. Also make sure you end lines with `;` in JS. If you don't the interpreter guesses where the line of code ends which can lead to some odd bugs